public/vislib/_map.js

```diff
@@ -441,46 +449,34 @@ define(function (require) {
       return isSame;
     };
 
-    TileMapMap.prototype.redrawDefaultMapLayers = function (url, options, enabled) {
+    TileMapMap.prototype.createBaseLayer = async function (getTileMapFromInvestigateYaml = false, url, options, enabled) {
```

Comment:
```suggestion
    TileMapMap.prototype.createBaseLayer = async function (getTileMapFromInvestigateYaml = null, url, options, enabled) {
```
Can you make it null instead? It seems like `getTileMapFromInvestigateYaml` is a boolean flag otherwise.